UPGRADE.md

```diff
@@ -0,0 +1,139 @@
+### Kubevirt HCO upgrade procedure
+
+Please note that the instructions were tested w/ OpenShift 4.2.
+Known issues related to the upgrade:
+
+[https://docs.google.com/document/d/1BM8NudsavT4Yh0Mvqv0D-vpqufYhWKAPradphbBB5KQ/edit?usp=sharing](https://docs.google.com/document/d/1BM8NudsavT4Yh0Mvqv0D-vpqufYhWKAPradphbBB5KQ/edit?usp=sharing "https://docs.google.com/document/d/1BM8NudsavT4Yh0Mvqv0D-vpqufYhWKAPradphbBB5KQ/edit?usp=sharing")
+
+
+###### 1. Start OpenShift 4.2 environment and make sure the KUBECONFIG is properly exported to point to the correct configuration files, so that kubectl command works properly.
```

Comment:
The assumption is that the user has already installed a cluster and CNV.  Let's only mention the steps it takes to go from an deployed release to a new one.

Reply:
OK, will update the document.

Reply:
Updated. Should we remove the manual flow as well (leaving only automatic one)?